openapi/api.yaml

```
@@ -0,0 +1,41 @@
+openapi: 3.0.2
+
+info:
+  title: move.mil API
+  description: The API for move.mil
+  #  termsOfService:
+  contact:
+    email: dp3@truss.works
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+  version: 0.0.1
+
+servers:
+  - url: http://api.move.mil/v1
+    description: Production server
+  - url: http://staging-api.move.mil/v1
+    description: Staging server
+  - url: http://experimental-api.move.mil/v1
+    description: Experimental server
+
+paths:
+  # Move Orders Endpoints
+  /move-orders:
+    $ref: './paths/move-orders.yaml#/MoveOrders'
+  /move-orders/{id}:
+    $ref: './paths/move-orders.yaml#/MoveOrderItem'
+  /move-orders/{id}/status:
```

```suggestion
  /move-orders/{moveOrderID}/status:
```

Let's always qualify IDs in paths so that the naming is consistent when we nest resources.